bofire/data_models/surrogates/mixed_single_task_gp.py

```diff
@@ -11,6 +11,11 @@
 )
 from bofire.data_models.surrogates.trainable_botorch import TrainableBotorchSurrogate
 
+from bofire.data_models.priors.api import (
```

Comment:
This import block is unsorted. You can just install the ruff extension in vscode and it will do it for you on save.